pom.xml

```diff
@@ -718,7 +724,7 @@
             <plugin>
                 <!-- https://stackoverflow.com/questions/46177921/how-to-run-unit-tests-in-excludedgroups-in-maven -->
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
+                <version>3.0.0-M1</version>
```

Comment:
Any particular reason for using higher plugin version?

M1 suffix points that this version could be not so stable as `2.22.x`

Reply:
It was due to a bug in OpenJdk8 debian distro + a bug in surefire plugin. I was unable to complete maven clean install due to build failing. I found on stackoverflow that it was fixed in M1 version. Another solution would be to change Java to Oracle distro which I'm using atm, but anyone using OpenJdk on debian could run into this problem.

Reply:
Ok. If earlier version is making problems then we can leave `3.0.0-M1` version